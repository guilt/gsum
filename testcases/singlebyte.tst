Name: singlebyte
Description: Hashing a file with a single byte (0x01)
File: singlebyte.bin 1B fill 01
AlgoMatrix: default
