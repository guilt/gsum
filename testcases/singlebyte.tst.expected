Hash-default: 4bf5122f344554c53e3cecd3d91803c1b6e08a5e3a6bcdcd5e8b2e6a6cfa1b48
